vmdb/app/helpers/miq_ae_class_helper.rb

```diff
@@ -15,4 +16,12 @@ def domain_display_name_using_name(record, current_domain_name)
       return domain_name, record.id
     end
   end
+
+  def automate_node?(treenodeid)
+    return true if treenodeid.start_with?('aen-')
```

Comment:
seems this could be turned into a single line?

```
treenodeid =~ /^ae[ncim]-/
```

OR

```
%w(aen- aec- aei- aem-).detect { |prefix| treenodeid.start_with?(prefix) }
```

Think the regex would be quicker in this case, but up to you for readability